docs/introduction/start-building-your-application.md

```diff
@@ -113,6 +113,24 @@ switch ($domainId) {
     case 3: $defaultCurrency = $this->getReference(CurrencyDataFixture::CURRENCY_CZK); break;
     ...
 ```
+
+## Fine-tune your configuration
+If all developers working on your project use the same version of PHP (eg. because all use SSFW via Docker), you can use higher versions of the libraries and tools installed via Composer.
+To do so, remove the `config.platform.php` option from your `composer.json`:
+```diff
+     "config": {
+         "preferred-install": "dist",
+         "component-dir": "project-base/web/components",
+-        "platform": {
+-            "php": "7.2"
+-        }
+     },
+```
+Run `composer update` to install updated versions of your dependencies (versions that don't support the lowest PHP version supported by SSFW).
```

Comment:
Again using SSFW

```suggestion
Run `composer update` to install updated versions of your dependencies (versions that don't support the lowest PHP version supported by Shopsys Framework).
```

Reply:
See above in https://github.com/shopsys/shopsys/pull/1352#discussion_r317626167